skipchain/struct.go

```diff
@@ -477,6 +451,60 @@ func (sb *SkipBlock) GetForwardLen() int {
 	return len(sb.ForwardLink)
 }
 
+// SignatureProtocol returns the name of the byzcoinx protocols that should
+// be used to sign forward links coming from this block
```

Comment:
can you update the comment here to say the first return value is the name of the new block protocol and the second is the follow block protocol?